stream_chat.go

```diff
@@ -1,7 +1,7 @@
 //go:generate go run github.com/getstream/easyjson/easyjson -pkg -all
 
-// Package stream_chat provides chat via stream API
-package stream_chat // nolint: golint
+// package stream_chat //nolint:golint provides "chat as an API" via stream
```

Comment:
something is wrong here

Reply:
Ohh good catch, not sure why go test didn't fail catch this.